### PR TITLE
Adds a Shakespeare reference to skulls

### PR DESCRIPTION
**Pull request by F-OS**

### CODE REVIEW

File: code/obj/item/organ.dm [hunk 1]
```diff
@@ -1296,6 +1296,25 @@
...
 
+/obj/item/skull/attack_self(mob/user as mob)
+	var/nerdlist = list()
+	for (var/mob/living/carbon/M in range(7, user))
+		if (M == user || !istype(user.loc, /turf)) 
+			continue
+		nerdlist += M
+	user.visible_message("<span style=\"color:blue\">[user] holds out [src] and stares into it.</span>")
+	if(src.donor || src.donor_name)
+		if(length(nerdlist) != 0)
```
Comment (ZeWaka) [inline code/obj/item/organ.dm]: Could squash this and the one below to a ternary

File: code/obj/item/organ.dm [hunk 2]
```diff
@@ -1296,6 +1296,25 @@
 		else
 			return ..()
 
+/obj/item/skull/attack_self(mob/user as mob)
```
Comment (ZeWaka) [inline code/obj/item/organ.dm:1300]: I would have an interaction limiter so people can't spam this constantly. Might be built into items, but I doubt it.

File: code/obj/item/organ.dm [hunk 3]
```diff
@@ -1296,6 +1296,25 @@
 		else
 			return ..()
 
+/obj/item/skull/attack_self(mob/user as mob)
+	var/nerdlist = list()
+	for (var/mob/living/carbon/M in range(7, user))
```
Comment (ZeWaka) [inline code/obj/item/organ.dm]: orange() includes current tile, which might be nice

Comment (Iamgoofball) [reply code/obj/item/organ.dm]: other way around, orange explicitly excludes the origin tile

Comment (ZeWaka) [reply code/obj/item/organ.dm]: @iamgoofball darn, looks like erik was wrong when he said that then

Comment (Iamgoofball) [reply code/obj/item/organ.dm]: additionally it'd be more performant to loop a list of all humans and check dist on 'em than it is to loop an entire range(7)

Comment (Iamgoofball) [reply code/obj/item/organ.dm]: additionally it'd be more performant to loop a list of all humans and check dist on 'em than it is to loop an entire range(7)

File: code/obj/item/organ.dm (hunk 2 above)
Comment (F-OS) [reply code/obj/item/organ.dm:1300]: Added a spamcheck

File: code/obj/item/organ.dm (hunk 3 above)
Comment (F-OS) [reply code/obj/item/organ.dm]: Goddamnit!!! I'll fix that.

Comment (ZeWaka) [reply code/obj/item/organ.dm]: > 
> 
> additionally it'd be more performant to loop a list of all humans and check dist on 'em than it is to loop an entire range(7)

Goon only has global lists of mobs and clients, sadly.

Comment (F-OS) [reply code/obj/item/organ.dm]: Corrected to orange, works correctly even if someone's on the same tile.

Comment (Iamgoofball) [reply code/obj/item/organ.dm]: Then make a humans global tbh

**Pull request closed without merging**